test_cfg/settings.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': ':memory:',
+    }
+}
+
+INSTALLED_APPS = (
+    'django.contrib.admin',
+    'django.contrib.sites',
+    'django.contrib.contenttypes',
+    'django.contrib.auth',
+    'django.contrib.messages',
+    'cms',
+    'menus',
+    'treebeard',
+    'git_md_page',
+)
+
+SECRET_KEY = 'TOP SECRET'
+
+BASE_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', 'git_md_page')
+TEMPLATES = [
+    {
+        'APP_DIRS': True,
+        # 'DIRS': [os.path.join(BASE_DIR, 'templates')],
```

Comment:
Is there any particular reason for commenting this line out?

Reply:
Removed.

Reply:
Resolved.